new-app/src/App/runGame/InputGuess.tsx

```diff
@@ -26,19 +26,29 @@ const InputGuess = () => {
   };
 
   return (
-    <div>
-      <input
-        value={guessString}
-        onChange={handleInput}
-        placeholder="Enter a 4-digit number"
-        className="input"
-        maxLength={4}
-        onKeyPress={handleOnKeyPress}
-      />
-      <button onClick={handleOnClick}>Submit</button>
-      <p>Press Enter to submit a guess</p>
-      <p>Your Guess: {printGuess}</p>
-    </div>
+    <body className="App">
+      <div className="bg-purple-200">
+        <h2>Real Guess Input</h2>
+        <div>
+          <input
+            value={guessString}
+            onChange={handleInput}
+            placeholder="Enter a 4-digit number"
+            className="input"
+            maxLength={4}
```

Comment:
This feels like a magic number. Thoughts on pulling this out? Could be something like `Code.maxLength`? Or just in this file: `const GUESS_MAX_LENGTH = 4`.